test/WebSites/InlineConstraintsWebSite/Constraints/IsbnDigitScheme10Constraint.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Routing;
+
+namespace InlineConstraintsWebSite.Constraints
+{
+    public class IsbnDigitScheme10Constraint : IRouteConstraint
+    {
+        private readonly bool _turnOn;
+
+        public IsbnDigitScheme10Constraint(bool turnOn)
```

Comment:
Maybe I missed something, but what exactly is this `turnOn` parameter?

I think I was envisioning something different: a single `isbn` constraint that takes a parameter of either 10 or 13 specifying the format. I don't think we need two different constraints.